src/main/java/edu/kis/powp/jobs2d/commands/factories/FiguresJoeToCommandFactory.java

```diff
@@ -0,0 +1,53 @@
+package edu.kis.powp.jobs2d.commands.factories;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.commands.ComplexCommand;
+import edu.kis.powp.jobs2d.commands.DriverCommand;
+import edu.kis.powp.jobs2d.magicpresets.FiguresJoe;
+
+import java.util.ArrayList;
+
+import static edu.kis.powp.jobs2d.commands.utils.Constants.*;
+
+public class FiguresJoeToCommandFactory implements Job2dDriver {
```

Comment:
Actually, it would be very easy to convert this solution to a single generic functionality (to cover building command from any script). Actually it is even possible now, because we can externally execute any set of commands on FiguresJoeToFactoryCommand (but the name doesn't suggest such possibility).